virtwho/virt/libvirtd/libvirtd.py

```diff
@@ -375,7 +375,7 @@ def _listDomains(self):
                     continue
 
                 if domain.UUIDString() == "00000000-0000-0000-0000-000000000000":
-                    # Don't send Domain-0 on xen (zeroed uuid)
+                    # Don't send Domain-0 (zeroed uuid)
```

Comment:
Could be this zeroed uuid set in any case and should we ignore it? I don't understand why is this code (related to xen) in `libvirtd.py`.

Reply:
I do not know why this is in there as this code is quite old. I left in the condition so that it would still behave the same way. It may never occur now that Xen is gone, but I wanted to be safe.